dm/syncer/data_validator.go

```diff
@@ -387,7 +391,7 @@ func (v *DataValidator) printStatusRoutine() {
 			speed := float64((currProcessedBinlogSize-prevProcessedBinlogSize)>>20) / interval.Seconds()
 			prevProcessedBinlogSize = currProcessedBinlogSize
 			prevTime = currTime
-
+			v.vmetric.ValidatorErrorCount.Set(float64(v.newErrorRowCount.Load()))
```

Comment:
this fields only stores in-memory new errors, we should load error count using getAllErrorCount which maybe slow that's why we put it in this routine.